.circleci/config.yml

```diff
@@ -426,7 +427,111 @@ jobs:
           comment: "Failed to Clean up prnet"
           pr_number: << pipeline.parameters.pr_number >>
 
-  build-and-push-docker-image:
+  build-helm-chart:
+    environment:
+      - IMAGE: taraxa-node
+      - HELMCHART: taraxa-node
+    machine:
+      image: ubuntu-2204:2022.04.2
+      docker_layer_caching: true
+    resource_class: medium
+    steps:
+      - checkout
+
+      - run:
+          name: Prepare Environment
+          command: |
+            echo "export PR=$( [[ -z ${CIRCLE_PULL_REQUEST+x} ]] && echo false || echo $(basename ${CIRCLE_PULL_REQUEST}) )" >> $BASH_ENV
```

Comment:
Don't really like this repeating part where we set env vars. Maybe we can create a yaml ref for this.

Reply:
Moved to separate `command` of Circle Ci, in which are :
- defined env variable for later tasks
- installed tools (`jq`, `yq`, ...)
- cleaning old images, ...